spec/features/ability_spec.rb

```diff
@@ -57,12 +56,30 @@
     expect(page).to have_link('Questions', href: "/admin/conferences/#{conference1.short_title}/questions")
     expect(page).to have_link('Roles', href: "/admin/conferences/#{conference1.short_title}/roles")
     expect(page).to have_link('Resources', href: "/admin/conferences/#{conference1.short_title}/resources")
+    expect(page).to have_link('Splashpage', href: "/admin/conferences/#{conference1.short_title}/splashpage")
+    expect(page).to have_link('Reports', href: "/admin/conferences/#{conference1.short_title}/program/reports")
+    expect(page).to have_link('Registration Period', href: "/admin/conferences/#{conference1.short_title}/registration_period")
 
     visit edit_admin_conference_path(conference1.short_title)
     expect(current_path).to eq(edit_admin_conference_path(conference1.short_title))
 
-    visit admin_conference_path(conference1.short_title)
```

Comment:
Because I think It is already being tested here: https://github.com/shlok007/osem/blob/01deb0c3e5fe1ca99a226e5ae5593da3509645f9/spec/features/ability_spec.rb#L32 .